src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -321,16 +321,24 @@ public ImageIcon() {
      */
     protected void loadImage(Image image) {
         MediaTracker mTracker = getTracker();
+        Boolean bIsInterrupted = false;
```

Comment:
Why do you use object wrapper instead of the primitive `boolean` type?

In Java, you don't add type prefixes to variable names; `bIsInterrupted` should be `interrupted`.